tests/e2e/main_test.go

```diff
@@ -191,3 +193,53 @@ func getLabelsDocumentation() (map[string][]string, error) {
 	}
 	return documentedMetrics, nil
 }
+
+func TestTelemetryMetrics(t *testing.T) {
```

Comment:
When is this test called, are we sure this is called right after everything is run? 🤔

Reply:
@lilic No, it is not!
That's what I explained in the PR's first message, but I was perhaps not clear 😊. I was asking you if would prefer if I called this test after each other test...

Reply:
Sorry missed that! At the end would be good, as this metrics only increases we should be able to see any error at the end. 

We seem to run the tests not at the very end, so I guess we need to run the go tests at the end or rethink the bash script? @tariq1890 
https://github.com/kubernetes/kube-state-metrics/blob/master/tests/e2e.sh#L180

Reply:
Yes, we could move go tests execution later. However, as e2e tests are re-written to Go (for example with #950), the bash tests currently executed after the Go ones will disappear.

Reply:
yep maybe we should merge that first, good point!